frontend/src/static/js/v_summary_charts.js

```diff
@@ -266,6 +266,22 @@ window.vSummaryCharts = {
       }
       return (Math.round((index + 1) * 1000 / this.filtered.length) / 10).toFixed(1);
     },
+
+    getGroupName(group) {
+      return window.getGroupName(group, this.filterGroupSelection);
+    },
+
+    isGroupMerged(groupName) {
+      return this.mergedGroups.includes(groupName);
+    },
+
+    handleMergeGroup(groupName) {
+      this.$emit('handle-merge-group', groupName);
+    },
+
+    handleExpandGroup(groupName) {
+      this.$emit('handle-expand-group', groupName);
+    },
```

Comment:
we now implemented vuex for event emitting, can try use vuex here?

Reply:
But right now `mergedGroups` is a property of `vSummary`. Do you mean making it as a global variable in `Vuex.Store`?

Reply:
You can make the `groupName` to global variable and watch it in vSummary. I am not very sure about this though as you are only emitting this to the direct parent. @fzdy1914 do you feel we should handle all event emit in Vuex